src/components/hooks/use-user-browser.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { browserName, browserVersion } from 'react-device-detect'
```

Comment:
Please get this library verified from the security team.

Reply:
It's done.